Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1789
+@@||csm.cxpublic.com^|
```

Comment:
It is related to cxense.com widget.
Better fix like https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/afd297abef3d55092f066f5404b73aad6bb7e963/Filters/exclusions.txt#L816

Reply:
Ok, changed it.